it/connectors/IncomeTaxUserDataConnectorSpec.scala

```diff
@@ -50,10 +50,10 @@ class IncomeTaxUserDataConnectorSpec extends IntegrationTest{
       "submission returns a 200" in {
 
         stubGetWithHeadersCheck(s"/income-tax-submission-service/income-tax/nino/$nino/sources/session\\?taxYear=$taxYear", OK,
-          Json.toJson(userData(employmentsModel)).toString(), ("X-Session-ID" -> sessionId), ("mtditid" -> mtditid))
+          Json.toJson(userData(fullEmploymentsModel(None))).toString(), ("X-Session-ID" -> sessionId), ("mtditid" -> mtditid))
```

Comment:
the brackets around "X-Session ID -> sessionId" and "mtditid -> mtditid" can be removed